lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -81,6 +81,12 @@
     - This is required if C(name) is not supplied.
     - If virtual machine does not exists, then this parameter is ignored.
     - Please note that a supplied UUID will be ignored on virtual machine creation, as VMware creates the UUID internally.
+  use_instance_uuid:
+    description:
+    - Use the VMWare instance UUID rather than the BIOS UUID.
+    default: 'no'
```

Comment:
Boolean values can now simply be yes or no. Please do this for all occurrences.
```suggestion
    default: no
```

Reply:
Please close comments that you have implemented. Outdated means the line has been changed, but doesn't guarantee that the comment was implemented. (In this case it's likely).

So please "resolve" implemented comments so it's clear what the status is of a review.